code/modules/antagonists/heretic/magic/void_prison.dm

```diff
@@ -0,0 +1,87 @@
+/datum/action/cooldown/spell/pointed/void_prison
+	name = "Void Prison"
+	desc = "Sends a heathen into the void for 10 seconds. \
+		They will be unable to perform any actions for the duration. \
+		Afterwards, they will be chilled and returned to the mortal plane."
+	background_icon_state = "bg_heretic"
+	overlay_icon_state = "bg_heretic_border"
+	button_icon = 'icons/mob/actions/actions_ecult.dmi'
+	button_icon_state = "voidball"
+	ranged_mousepointer = 'icons/effects/mouse_pointers/throw_target.dmi'
+	sound = 'sound/magic/voidblink.ogg'
+
+	cooldown_time = 1 MINUTES
+
+	sound = null
+	school = SCHOOL_FORBIDDEN
+	invocation = "V''D PR'S'N!"
+	invocation_type = INVOCATION_SHOUT
+	spell_requirements = NONE
+
+/datum/action/cooldown/spell/pointed/void_prison/cast(mob/living/carbon/human/cast_on)
+	. = ..()
+	if(cast_on.can_block_magic(antimagic_flags))
+		cast_on.visible_message(
+			span_danger("[cast_on]'s flashes in a firey glow, but repels the blaze!"),
+			span_danger("Your body begins to flash a firey glow, but you are protected!!")
+		)
+		return
+	cast_on.apply_status_effect(/datum/status_effect/void_prison, "void_stasis")
+
+/datum/status_effect/void_prison
+	duration = 10 SECONDS
```

Comment:
I believe you need to set `alert_type = null` or it gives the default one

Also you need to set `id`